.env

```diff
@@ -4,4 +4,5 @@ REACT_APP_WMNDS_VERSION = 1.7.3
 
 # Variables that control the site title and meta descriptions
 REACT_APP_TITLE = "Disruptions"
+REACT_APP_HEADER_TITLE = "Find live and planned disruptions"
```

Comment:
Where is this being used? You can change the REACT_APP_TITLE instead

Reply:
Used in index.html, line 141 as the header line. REACT_APP_TITLE was already used in multiple places within the project so I can't amend the existing text. REACT_APP_HEADER_TITLE is in addition to the REACT_APP_TITLE